### PR TITLE
[8]Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `Added navigation to the pages
- **`Contact page**
contact-form

**Sign up page**
signup-page

**Product Page**
product-page

### CODE REVIEW

File: submissions/chabuChabu/html-forms/contact-form.html [hunk 1]
```diff
@@ -20,7 +20,11 @@ <h4>Thank you for getting in touch. Please fill out your details and type your i
             <input type="email" name="email" id="email" ><br><br>
             <label><strong>Please type your inquiry below</strong></label><br><br>
             <label>Subject:</label><br>
+<<<<<<< HEAD
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-forms/contact-form.html:23]: Remove this line

File: submissions/chabuChabu/html-forms/contact-form.html [hunk 2]
```diff
@@ -20,7 +20,11 @@ <h4>Thank you for getting in touch. Please fill out your details and type your i
...
+            <input type="text" name="subject" id="subject" ><br><br>
+=======
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-forms/contact-form.html:25]: Remove this line

File: submissions/chabuChabu/html-forms/contact-form.html [hunk 3]
```diff
@@ -20,7 +20,11 @@ <h4>Thank you for getting in touch. Please fill out your details and type your i
...
             <input type="text" name="subject" id="subject"><br><br>
+>>>>>>> cae1c79acf7037b12df3615319fba954bcb0681f
```
Comment (ToluAkin) [inline submissions/chabuChabu/html-forms/contact-form.html:27]: Remove this line

File: submissions/chabuChabu/html-forms/contact-form.html (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html-forms/contact-form.html]: Done

File: submissions/chabuChabu/html-forms/contact-form.html (hunk 2 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/html-forms/contact-form.html]: Done

**Comment by ChabuChabu** [comment]
> @ChabuChabu After remove the lines, sync your branch, that should fix the conflicts on your pull request. I noticed that you deleted the `product.html` file in `html` folder, does another `product.html` exist in `introToHTML`?

@ToluAkin Yes i have one in introToHTML,so i removed it from html folder

**Comment by ChabuChabu** [comment]
> @ChabuChabu After remove the lines, sync your branch, that should fix the conflicts on your pull request. I noticed that you deleted the `product.html` file in `html` folder, does another `product.html` exist in `introToHTML`?

**Pull request merged**